sympy/parsing/c/c_parser.py

```diff
@@ -1033,6 +1033,30 @@ def get_expr_for_operand(self, combined_variable):
             if combined_variable[1] == 'boolean':
                     return true if combined_variable[0] == 'true' else false
 
+        def transform_while_stmt(self, node):
+            """Transformation function for handling while statement
+
+            Returns
+            =======
+
+            statement : Codegen AST Node
+                contains the while statement node
```

Comment:
can you add a little bit more explanation, like it returns a while node, with the condition and statement children node, something like that.

Reply:
Sure :)